sdk/communication/azure-communication-rooms/src/main/java/com/azure/communication/rooms/models/CreateRoomOptions.java

```diff
@@ -26,6 +26,12 @@ public final class CreateRoomOptions {
      */
     private OffsetDateTime validUntil;
 
+    /*
+     * Set this flag to true if, at the time of the call, dial out to a PSTN number is enabled in a particular room. By
+     * default, this flag is set to false.
+     */
+    private Boolean pstnDialOutEnabled = false;
```

Comment:
This should be `null`. The service should set the default and this should not be set in the client.

Reply:
Removed default value